docs/terraform_loadtest.md

```diff
@@ -52,8 +52,11 @@ go run ./cmd/ltctl deployment create
 ```
 
 This command can take several minutes to complete when creating a [full](loadtest_system.md) deployment. By default, the console will keep logging info messages. If it does not, then something's wrong.
+
 Once done, it will output information about the entire cluster. Everything will be now ready to start a new load-test.
 
+If you hit an error here that a value already exists, it is most likely because your `ClusterName` is not a unique value. Change that value and create the deployment again. You shoudl run `go run ./cmd/ltctl deployment destroy` to clean up the half created deployment.
```

Comment:
Maybe the whole thing should be reworded, we both said the same thing. What do you think about the below?

> If you see an error when running `deployment create` mentioning a "resource already exists", it is most likely because your `ClusterName` is not a unique value within your AWS account. Run `go run ./cmd/ltctl deployment destroy`  to clean up the half created deployment. Then change the `ClusterName` to something more unique for your loadtest and try again.

Reply:
My bad... yes we were saying the same thing.  It read differently in my head the first time.  Your suggestion makes sense.

Reply:
The suggestion in your comment, @coltoneshaw, makes sense to me, and is clearer than the original in my opinion.